sdks/java/core/src/main/java/org/apache/beam/sdk/state/StateContext.java

```diff
@@ -17,13 +17,16 @@
  */
 package org.apache.beam.sdk.state;
 
+import org.apache.beam.sdk.annotations.Internal;
 import org.apache.beam.sdk.options.PipelineOptions;
 import org.apache.beam.sdk.transforms.windowing.BoundedWindow;
 import org.apache.beam.sdk.values.PCollectionView;
 
 /**
- * Information accessible the state API.
+ * For internal use only; no backwards-compatibility guarantees.
+ * <b><i>For internal use only; no backwards-compatibility guarantees.</i></b>
```

Comment:
there are more. happy for you to decide whether you want warning + old javadoc or just warning , which is currently slightly inconsistent. Up to you completely perccase.